versioned_docs/version-0.2.0/contribute-to-kubeslice-docs.md

```diff
@@ -0,0 +1,52 @@
+# Contribute to KubeSlice documentation
+
+_KubeSlice welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to the KubeSlice documentation by
+
+- Improving existing content
+- Suggesting new content
+
+## Getting Started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or report a bug/typo with the help of a GitHub issue to the
+[docs](https://github.com/kubeslice/docs) repository.
+
+## Pre-requisites 
+
+Below is the list of things you should be aware of before getting started with your contribution journey to the KubeSlice documentation
+
+- [Git](https://git-scm.com/)
+- [GitHub](https://lab.github.com/)
+- [Docusaurus](https://docusaurus.io)
+    - Docusaurus is a static site generator that makes use of the Markdown syntax and uses [MDX](https://mdxjs.com/) as the parser.
+
+## Making your first contribution!
+
+- Familiarize yourself with the [GitHub repository](https://github.com/kubeslice/docs) hosting the documentation.
+- Whether you’re suggesting an improvement or adding new content, you will need to open an issue first.
+	- Refer to our section on opening an issue below to learn more.
+
+
+## Opening an issue
```

Comment:
Please change it to title case as - `Opening an Issue`

Reply:
@veenamj-avesha , I'll go ahead and merge the PR as it has been open for a while. Please create another PR if the title case needs to be implemented.